package.json

```diff
@@ -28,25 +28,25 @@
 	],
 	"scripts": {
 		"lint": "eslint src --ext ts --fix",
-		"format": "prettier --write src/**/*.ts",
+		"format": "prettier --write \"src/**/*.ts\"",
 		"update": "yarn up \"*\" -i",
 		"build": "tsc -b src",
 		"clean": "tsc -b src --clean",
 		"watch": "tsc -b src -w",
-		"sversion": "standard-version -a",
-		"commit": "git-cz",
-		"cz": "git-cz",
+		"sversion": "standard-version",
 		"prepublishOnly": "yarn build",
 		"prepare": "husky install .github/husky"
 	},
 	"dependencies": {
-		"chalk": "^4.1.2",
+		"@sapphire/result": "^1.0.0",
+		"colorette": "^2.0.16",
 		"commander": "^9.0.0",
+		"execa": "^6.1.0",
 		"find-up": "^5.0.0",
-		"ora": "^6.0.1",
+		"js-yaml": "^4.1.0",
+		"ora": "^6.1.0",
```

Comment:
have u tried nanospinner?
> Only single dependency (picocolors).
It 45 times smaller than ora.
Support both CJS and ESM projects.
TypeScript type declarations included.

Reply:
I'll look into it

Reply:
Ended up completely porting over that lib's code because I wanted to use colorette instead of picocolor so to use only 1 coloring library across the code base. Also wanted to have it use a class instead a function

Reply:
> Ended up completely porting over that lib's code because I wanted to use colorette instead of picocolor so to use only 1 coloring library across the code base. Also wanted to have it use a class instead a function

have you considered just using picocolor instead?